ignite/handlers/lr_finder.py

```diff
@@ -498,4 +510,4 @@ def __init__(self, optimizer: Optimizer, start_lr: List[float], end_lr: float, n
     def get_lr(self) -> List[float]:  # type: ignore
         curr_iter = self.last_epoch + 1
         r = curr_iter / self.num_iter
-        return [base_lr * (self.end_lr / base_lr) ** r for base_lr in self.base_lrs]
+        return [base_lr * (end_lr / base_lr) ** r for end_lr, base_lr in zip(self.end_lr, self.base_lrs)]
```

Comment:
@Jacob208M you need to update type hints in the constructor:
```diff
- end_lr: float
+ end_lr: List[float]
```

Reply:
I did it.